README.md

```diff
@@ -147,7 +152,7 @@ While your workspace itself is a clone of [MovingBlocks/Terasology](https://gith
 Accordingly, if you want to contribute to modules, you'll need to navigate into the respective subdirectory and work with Git from in there.
 Any Git commands executed in your workspace root will target [MovingBlocks/Terasology](https://github.com/MovingBlocks/Terasology).
 
-For more information, see our wiki entry on [Understanding Terasology's Git Setup](https://terasology.org/Terasology/#/Developing-Modules?id=understanding-terasology39s-git-setup).
+See as well our wiki entry on [Understanding Terasology's Git Setup](https://terasology.org/Terasology/#/Developing-Modules?id=understanding-terasology39s-git-setup).
```

Comment:
Not sure what's wrong with the original here?
@BenjaminAmos as a native speaker, can you comment on what's "good English" to use here?